internals/secrethub/env_ls.go

```diff
@@ -23,7 +23,8 @@ func NewEnvListCommand(io ui.IO) *EnvListCommand {
 
 // Register adds a CommandClause and it's args and flags to a Registerer.
 func (cmd *EnvListCommand) Register(r command.Registerer) {
-	clause := r.Command("ls", "Read the value of a single environment variable.")
+	clause := r.Command("ls", "[BETA] List environment variable keys that will be injected with secrets.")
```

Comment:
How about: `List environment variable names that will be populated with secrets`